### ISSUE TITLE
Live demo broken?

### ISSUE THREAD

**Issue by lmaylein** [zxing-js/library#472]
https://zxing-js.github.io/library/examples/multi-camera/

needs

https://unpkg.com/@zxing/library@0.18.6/umd/zxing-browser.min.js

which returns a 404 error

**Comment by crozone** [comment]
Yeah, it need https://unpkg.com/@zxing/library@0.18.6/umd/index.min.js instead.
